index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Excursion</title>
+        <link href="../excursion/resources/css/style.css" rel="stylesheet">
+    </head>
+    <body>
+        <header>Discover hidden places in the world around you</header>
+        <main>
+            <p><a href="#">Download Excursion (Coming soon!)</a></p>
+            <section>
+                <img alt="iPhone with travel video clip">
```

Comment:
No image here.

Reply:
It's just a placeholder at this stage - work in progress 🔢